src/main/java/stream/flarebot/flarebot/commands/general/ServerInfoCommand.java

```diff
@@ -0,0 +1,117 @@
+import stream.flarebot.flarebot.FlareBot;
+import stream.flarebot.flarebot.commands.Command;
+import stream.flarebot.flarebot.commands.CommandType;
+import stream.flarebot.flarebot.objects.GuildWrapper;
+import stream.flarebot.flarebot.util.MessageUtils;
+
+import java.awt.Color;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.TimeUnit;
+
+public class ServerInfoCommand implements Command {
+
+    @Override
+    public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Message message, String[] args, Member member) {
+        if(args.length == 0){
+            sendGuildInfo(guild.getGuild(), channel);
+        } else {
+            String guildid = args[0];
+            Guild targetGuild = FlareBot.getInstance().getGuildByID(guildid);
+            if(targetGuild != null){
+                sendGuildInfo(targetGuild, channel);
+            } else {
+                MessageUtils.sendErrorMessage("We couldn't find that guild.", channel);
+            }
+        }
+    }
+
+    private void sendGuildInfo(Guild guild, TextChannel channel){
+        EmbedBuilder eb = new EmbedBuilder();
+        eb.setTitle(guild.getName());
+        eb.setThumbnail(guild.getIconUrl());
+        eb.addField("Users", "**Total:** " +
+                guild.getMembers().size() + "\n" +
+                "\n" +
+                "**Online:** " +
+                guild.getMembers().stream().filter(member -> !member.getOnlineStatus().equals(OnlineStatus.OFFLINE)).count() + "\n" +
+                "\n" +
+                "**Owner:** " +
+                guild.getOwner().getUser().getName() +  "#" + guild.getOwner().getUser().getDiscriminator(), true);
+        String afk = guild.getAfkChannel() == null ? "" :
+                "**AFK:**\n" +
+                        "Channel: " +
+                        guild.getAfkChannel().getName() + "\n" +
+                        "\n" +
+                        "Timeout: " +
+                        FlareBot.getInstance().formatTime(guild.getAfkTimeout().getSeconds(), TimeUnit.SECONDS, true, false);
+        eb.addField("Channels", "**Text**\n" +
+                "Total: " +
+                guild.getRoles().size() + "\n" +
+                "\n" +
+                "Default: " +
+                "#" + guild.getTextChannelById(guild.getId()).getName() + "\n" +
+                "\n" +
+                "**Voice**\n" +
+                "Total: " +
+                guild.getVoiceChannels().size() + "\n" +
+                "\n" +
+                "Active: " +
+                guild.getVoiceChannels().stream().filter(voiceChannel -> voiceChannel.getMembers().size() > 0).count() + "\n" +
+                "\n" + afk, true);
+        eb.addField("Misc info", "**Creation time:** " +
+                guild.getCreationTime().toLocalDateTime().atOffset(ZoneOffset.UTC).format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")) + " GMT/BST\n" +
```

Comment:
Timezones are a huge confusion. ~~Just ask Numnberphile or Computerphile, not sure which.~~

Use this formattter: `yyyy-MM-dd HH:mm:ss 'UTC'`